package.json

```diff
@@ -1,56 +1,81 @@
 {
   "name": "@lesechos/image-size-loader",
-  "version": "1.0.1",
-  "description": "An image size loader module for webpack",
+  "version": "2.0.0",
   "license": "Apache-2.0",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/LesEchos/image-size-loader"
-  },
+  "description": "An image size loader module for webpack",
+  "keywords": [
+    "webpack",
+    "assets",
+    "images",
+    "meta"
+  ],
   "author": "Léo Lapujade <leo.lapujade@gmail.com>",
   "contributors": [
     "Boopathi Rajaa <me@boopathi.in>",
     "Léo Lapujade <leo.lapujade@gmail.com>",
     "Patrick Collins <pat@burned.com>",
-    "Vignesh Shanmugam <vignesh.shanmugam22@gmail.com>",
-    "Tobias Koppers @sokra"
+    "Tobias Koppers @sokra",
+    "Vignesh Shanmugam <vignesh.shanmugam22@gmail.com>"
   ],
-  "bin": "",
-  "main": "dist/cjs.js",
+  "homepage": "https://github.com/LesEchos/image-size-loader",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/LesEchos/image-size-loader"
+  },
+  "bugs": {
+    "url": "https://github.com/LesEchos/image-size-loader/issues"
+  },
+  "main": "index.js",
+  "types": "dist",
   "engines": {
-    "node": ">= 6.9.0 < 7.0.0 || >= 8.9.0"
+    "node": ">=14",
+    "npm": ">=6"
   },
   "scripts": {
-    "start": "npm run build -- -w",
-    "build": "cross-env NODE_ENV=production babel src -d dist --copy-files",
-    "clean": "del-cli dist",
+    "build": "tsc --project 'tsconfig.build.json'",
+    "clean": "rimraf dist",
+    "lint": "concurrently -r -m 1 'npm:lint-*'",
+    "lint-js": "eslint '**/*.{js,jsx,ts,tsx}'",
+    "lint-ts": "tsc --project 'tsconfig.development.json'",
```

Comment:
isn't it easier to maintain only one tsconfig.json file, and use `tsc --noEmit` here?

Reply:
There is more than just `noEmit` option inside each `tsconfig` file.